server/lib/src/be/mod.rs

```diff
@@ -571,6 +571,10 @@ pub trait BackendTransaction {
                 filter_error!("Requested a top level or isolated AndNot, returning empty");
                 (IdList::Indexed(IDLBitRange::new()), FilterPlan::Invalid)
             }
+            FilterResolved::Invalid(_) => {
+                // Partial is the only one that doesn't run the risk of being cancelled due to resource constraint
```

Comment:
Stale comment. Perhaps "This is considered indexed as it can never evaluate to try and shouldn't influence filter testing"

Reply:
whoops, Thought I had deleted it :(